jobs/involuntary-dissolutions/Makefile

```diff
@@ -39,7 +39,7 @@ clean-test: ## clean test files
 build-req: clean ## Upgrade requirements
 	test -f venv/bin/activate || python3.8 -m venv  $(CURRENT_ABS_DIR)/venv ;\
 	. venv/bin/activate ;\
-	pip install --upgrade pip ;\
```

Comment:
I reverted the changes in Makefile back to previous version because it resolved the `ModuleNotFoundError: No module named 'involuntary_dissolutions'` I got when running tests.

Reply:
Hmm, weird. The `--upgrade` change [was a workaround for a dependency error](https://github.com/bcgov/lear/pull/2714#issuecomment-2135395063) that was happening on my mac (and maybe Argus' mac too), but obviously we don't want it breaking the unit tests.

I've been messing around with this a bit to see if I can find a middle ground where the requirements get installed correctly and the unit tests still run. I've found that `pip install pip==21.0` works for both cases but throws a bunch of deprecation warnings during the tests.

I plan to keep experimenting a bit but I will make the change in my PR if I find a version that works for both cases.